adds fill_nulls_with to all metric types (#4768)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/conversion-metrics.md

| `constant_properties` | List of constant properties.  | List | Optional |
| `base_property` | The property from the base semantic model that you want to hold constant.  | Entity or Dimension | Optional |
| `conversion_property` | The property from the conversion semantic model that you want to hold constant.  | Entity or Dimension | Optional |

> | `fill_nulls_with` | Set the value of a null conversion event to zero instead of null. | String | Optional |

Author: ```suggestion
| `fill_nulls_with` | Set the value in your metric definition instead of null (such as zero). | String | Optional |
```

Refer to [additional settings](#additional-settings) to learn how to customize conversion metrics with settings for null values, calculation type, and constant properties.

The following code example displays the complete specification for conversion metrics and details how they're applied:

```yaml
metrics:
  - name: The metric name # Required
    description: The metric description # Optional
    type: conversion # Required
    label: # Required
    type_params: # Required

>       fills_nulls_with: Set value to zero instead of null # Optional

Author: ```suggestion
      fills_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
```
